spec/castle_devise/integration/password_update_spec.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+RSpec.describe "Password update", type: :request do
```

Comment:
WDYT about renaming the spec files to `password_reset_requested_spec.rb` and `profile_update_spec.rb` to match the event names?

Reply:
I wanted to be more explicit with what we test there but in order to keep it consistent with other files, I will rename this. 👍